store/app/routers/email_signup.py

```diff
@@ -0,0 +1,63 @@
+"""This module defines the FastAPI routes for managing email sign-up tokens."""
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from pydantic import BaseModel, EmailStr
+
+from store.app.crud.email_signup import EmailSignUpCrud
+from store.app.utils.email import send_register_email
+
+email_signup_router = APIRouter()
+
+
+# Request Model
+class EmailSignUpRequest(BaseModel):
+    email: EmailStr
+
+
+# Response Models
+class EmailSignUpResponse(BaseModel):
+    message: str
+
+
+class GetTokenResponse(BaseModel):
+    id: str
+    email: str
+
+
+class DeleteTokenResponse(BaseModel):
+    message: str
+
+
+# POST: Create Signup Token
+@email_signup_router.post("/create/", response_model=EmailSignUpResponse)
+async def create_signup_token(data: EmailSignUpRequest) -> EmailSignUpResponse:
```

Comment:
prefer to use google-style comments

```suggestion
@email_signup_router.post("/create/", response_model=EmailSignUpResponse)
async def create_signup_token(data: EmailSignUpRequest) -> EmailSignUpResponse:
    """Creates a signup token and emails it to the user."""
```